OTNetworkTest.m

```diff
@@ -98,8 +99,11 @@ -(void)dispatchResultsToDelegateWithResult:(enum OTNetworkTestResult)result
              respondsToSelector:@selector(networkTestDidCompleteWithResult:
                                           error:)])
         {
-            // [_myAudioDevice setAudioPlayoutMute:NO];
-            // [OTAudioDeviceManager setAudioDevice:nil];
+            [_myAudioDevice setAudioPlayoutMute:NO];
+
+            // Properly release the OTAudioDevice to be reset next time around
```

Comment:
This is the line that was crashing the app, trying to send nil to method that was expecting type 'OTAudioDevice'